src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPACE_SEPARATOR = " ";
+    private static final String COMMA_SEPARATOR = ",";
+    private static final int OPERATION_TYPE_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final int SUPPLY_AMOUNT = 0;
+    private static final int BUY_AMOUNT = 1;
+    private static final int RESULT_VALUE = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("supply,").append(getData(fromFileName)[SUPPLY_AMOUNT])
```

Comment:
```suggestion
        StringBuilder builder = new StringBuilder()
        .append("supply,").append(getData(fromFileName)[SUPPLY_AMOUNT])
```

Reply:
let's refactor your code so that the public method is more concise - it should have just three method invocation, one to readFromFile, second method receives what we've read and generates report string, third method receives string, filename and write that string to the file it will create